src/app/static/src/js/components/runWorkflow/runWorkflowReport.vue

```diff
@@ -217,7 +219,8 @@ export default Vue.extend<Data, Methods, Computed, Props>({
             importedFile: null,
             reportsOrigin: session.getSelectedWorkflowReportSource() || "list",
             importFromCsvIsEnabled: switches.workFlowReport,
-            isImportedReports: false
+            isImportedReports: false,
+            inactiveOriginWorkflowReports: session.getInactiveOriginWorkflowReports() || []
```

Comment:
```suggestion
            deactivatedReports: session.getInactiveOriginWorkflowReports() || []
```

Reply:
I think I prefer `inactiveOrigin` to `deactivated` because it is the origin itself that isn't active... "deactivated reports" is a bit mysterious!

Reply:
I agree with Alex here if I get the casing vote - I'd prefer long-winded but precise to terse but ambiguous.